### DISCUSSION TITLE
Birdseye restream

### DISCUSSION THREAD

**Discussion by Tokn59** [blakeblackshear/frigate#6488] [General]
Hi, the documentation states : 

Birdseye RTSP restream can be enabled at birdseye -> restream and accessed at rtsp://<frigate_host>:8554/birdseye. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.

Unclear exactly how/where I have to enable restream for Birdseye. I do not see such an option in the configuration. In my instance Birdseye is active and working but how/where do I enable restream ?

Thank You !

**Comment by NickM-27** [comment]
That simply means

```yaml
birdseye:
  restream: true
```

**Reply by show2go** [reply]
Oh my...
would be great to have this in the documentation, for such people like me... ;)
Thank you very much

**Comment by Tokn59** [comment]
Super ! Thank You !

**Reply by lpdescamps** [reply]
i put a pull request to add the code
https://github.com/blakeblackshear/frigate/pull/8015
